internal/models/network.go

```diff
@@ -36,6 +36,9 @@ type CompressedPayload struct {
 type Device struct {
 	RpiUptimeSecs  int     `json:"rpiUptimeSecs,omitempty"`
 	BatteryVoltage float64 `json:"batteryVoltage,omitempty"`
+	Version        string  `json:"version,omitempty"`
+	IMEI           string  `json:"imei,omitempty"`
+	UnitId         string  `json:"unitId,omitempty"`
```

Comment:
@JamesReate , should we query imei somehow? Right now we do not have that information

Reply:
yes, we should. there is a `GetIMSI`, under commands, probably something similar for IMEI, we can look if not in docs

Reply:
added GetIMEI